cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanBaseActivity.java

```diff
@@ -504,7 +506,11 @@ public void onPreviewFrame(byte[] bytes, Camera camera) {
 
     @Override
     public void onClick(View view) {
-        if (mCamera != null && mFlashlightId == view.getId()) {
+        if (mEnterCardManuallyId == view.getId()) {
+            // this is essentially clicking the back button?
+            onBackPressed();
```

Comment:
The original request was to expose the click event to callers. Instead of using `onBackPressed()`, we should consider returning a different result to the calling activity.

Consider creating a method `onEnterCardManuallyPressed()` that looks something like this:
```
    protected void onEnterCardManuallyPressed() {
        Intent intent = new Intent();
        intent.putExtra(RESULT_CANCELLED_REASON, REASON_ENTER_CARD_MANUALLY);
        setResult(RESULT_CANCELLED, intent);
        finish();
    }
```